template.go

```diff
@@ -57,6 +59,13 @@ func readYAMLRequest(opts *Options) error {
 	}
 
 	base := filepath.Dir(opts.ROpts.YamlTemplate)
+	base, err = filepath.Abs(base)
```

Comment:
// Ensuring that the base directory is fully qualified. Otherwise, whether it is fully qualified depends on argv[0]. Must be fully qualified to be expressible as a `file:///` URL. Go’s URL parser does not recognize `file:path` as host-relative, not-CWD relative.